src/components/content-tab-import-export.tsx

```diff
@@ -1,6 +1,138 @@
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import { getIpcApi } from '../lib/get-ipc-api';
+import { ExportOptions } from '../lib/import-export/export/types';
+import { BackupArchiveInfo } from '../lib/import-export/import/types';
+
 interface ContentTabImportExportProps {
 	selectedSite: SiteDetails;
 }
-export function ContentTabImportExport( _props: ContentTabImportExportProps ) {
-	return null;
+
+export const ExportSite = ( {
+	selectedSite,
+	onExport,
+}: {
+	selectedSite: SiteDetails;
+	onExport: ( options: ExportOptions ) => Promise< void >;
+} ) => {
+	const onExportFullSite = async () => {
+		const path = await getIpcApi().showSaveAsDialog( {
+			title: __( 'Save backup file' ),
```

Comment:
Is the title supposed to be visible in the UI?

<img width="497" alt="Screenshot 2024-07-19 at 11 28 49 AM" src="https://github.com/user-attachments/assets/104ecfe4-5f0a-42e9-8097-46f6610db85e">

Reply:
> Is the title supposed to be visible in the UI?

I'm not sure about that. Other dialogs have it, and I added it in order to be "compatible". In Linux, none of them appears, so I cannot test it properly.

Reply:
It seems it's not supported on macOS.